frontend/packages/core-api-client/tsconfig.cjs.json

```diff
@@ -3,5 +3,10 @@
   "compilerOptions": {
     "module": "commonjs",
     "outDir": "lib/cjs",
-  }
+  },
+  "exclude": [
```

Comment:
just for information, why did you move this to the 'child' tsconfigs? did something not work?

Reply:
So I had an issue that vscode wasn't working well on the test files because they were excluded in the tsconfig. Removing the exlusion rule fixed the issue, but broke the build. So I moved it to the build specific configs.